### PR TITLE
Feat/inga/change profile image

### PR DESCRIPTION
**Pull request by ingatrifan**

### CODE REVIEW

**Comment by procateodor** [comment]
[Update user profile picture](https://trello.com/c/uLRNmaYm/33-update-user-profile-picture)

File: server.js [hunk 1]
```diff
@@ -59,6 +59,12 @@ const server = async () => {
     transports: [new transports.Http(httpTransportOptions)]
   })
 
+  cloudinary.config({
+    cloud_name: 'scheduleapp',
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_API_SECRET
+  })
```
Comment (adriangotca98) [inline server.js]: use config object here, in process.env we'll only have NODE_ENV, PORT, DB_URI and SECRET_KEY, nothing more.

File: controllers/userController.js [hunk 2]
```diff
@@ -155,3 +155,30 @@ exports.resetPassword = async (req, res) => {
...
+    const profileImage = {
+      id: result.public_id,
+      path: result.secure_url
+    }
+
+    await req.db.User.updateOne({ email }, { profileImage })
+
+    return res.json({
+      success: true
+    })
```
Comment (adriangotca98) [inline controllers/userController.js]: Add a message to this response so every 200 response have a generic template

**Review by adriangotca98 — CHANGES_REQUESTED** [review]

**Review by procateodor — CHANGES_REQUESTED** [review]
See @adriangotca  comments.

**Review by procateodor — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by ingatrifan** [comment]
Am dat acum pull de pe master nu imi arata ca as avea conflicte, imi zice already up to date.

**Review by procateodor — DISMISSED** [review]

**Review by procateodor — APPROVED** [review]

File: controllers/userController.js (hunk 2 above)
Comment (adriangotca98) [reply controllers/userController.js]: message and status to the 200 response, please

File: server.js (hunk 1 above)
Comment (adriangotca98) [reply server.js]: use config object, the process.env will only have PORT, NODE_ENV, DB_URI and SECRET_KEY. All other config data will be stored on the config object. Also, make sure that CLOUDINARY_API_KEY and CLOUDINARY_API_SECRET are present in the db

**Review by adriangotca98 — CHANGES_REQUESTED** [review]

**Review by procateodor — APPROVED** [review]

**Review by adriangotca98 — APPROVED** [review]

**Pull request merged**